src/auth/reset_password/reset_password.modal.js

```diff
@@ -0,0 +1,27 @@
+(function () {
+    'use strict';
+
+    angular.module('motech-auth')
+        .service('ResetPasswordModal', ResetPasswordModal);
+
+    ResetPasswordModal.$inject = ['$compile', '$rootScope', 'ModalWindow'];
+    function ResetPasswordModal ($compile, $rootScope, ModalWindow) {
+        var modal;
+        return {
+            open: function () {
+                modal = ModalWindow(
+                    $compile('<reset-password />')($rootScope),
+                    $compile('<span>{{ "security.enterEmailQuestions" | translate}}</span>')($rootScope),
```

Comment:
Let's change the modal title from "Cant sign in? Forgot your password?" to "Reset Instructions" as it was previously made in Motech. It doesn't look good to repeat the same words at the beginning of the modal content and in the modal title.